extensions/wikia/ArchiveWikiForum/ArchiveWikiForumHooks.class.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+class ArchiveWikiForumHooks {
+
+    /**
+     * Show the notification informing the user that wiki forums have been disabled
+     * @param Article $article
+     * @param $outputDone
+     * @param $useParserCache
+     * @return bool
+     */
+    public static function onArticleViewHeader( &$article, &$outputDone, &$useParserCache ) {
+        $title = $article->getTitle();
+        if ( self::isForumNS( $title->getNamespace() ) ) {
+            $app = F::app();
+            $html = $app->renderView(
+                ArchiveWikiForumController::class,
+                ArchiveWikiForumController::ARCHIVED_FORUM_NOTIFICATION
+            );
+            $app->wg->Out->addHTML( $html );
+        }
+        return true;
+    }
+
+    /**
+     * Override button on forum
+     * @param WikiaResponse $response
+     * @param $ns
+     * @param $skin
+     * @return bool
+     */
+    public static function onPageHeaderIndexAfterActionButtonPrepared( $response, $ns, $skin ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameters "$ns, $skin". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)